features/step_definitions/filtering_steps.rb

```diff
@@ -558,14 +560,18 @@
   expect(page).to have_select('order', selected: "Relevance")
 end
 
-Then(/^I see updated newest order selected$/) do
-  expect(page).to have_select('order', selected: "Updated (newest)")
-end
+# Then(/^I see updated newest order selected$/) do
```

Comment:
Can we remove this?

Reply:
Good spot. I amended the previous commit.